plugins/modules/network/radware/vdirect_runnable.py

```diff
@@ -258,7 +258,7 @@ def _validate_action_name(self):
                 raise WrongActionNameException(self.action_name, available_actions)
 
     def _validate_required_action_params(self):
-        action_params_names = [n for n in self.action_params]
+        action_params_names = self.action_params
```

Comment:
```suggestion
        action_params_names = list(self.action_params)
```
`self.action_params` is a dictionary.